utils/python/tests/scripts_regression_tests.py

```diff
@@ -1115,11 +1115,15 @@ def simple_test(self, manual_timing=False):
     ###########################################################################
     def test_save_timings(self):
     ###########################################################################
+        if CIME.utils.get_model() != "acme":
```

Comment:
I think both of us need to be able to save timings. Was this test causing problems?